src/Diagnostics.DataProviders/DataProviderLogDecorator.cs

```diff
@@ -68,6 +68,11 @@ public Task<DataTable> ExecuteClusterQuery(string query, string requestId = null
             return ExecuteQuery(query, DataProviderConstants.FakeStampForAnalyticsCluster, requestId, operationName);
         }
 
+        public Task<KustoQuery> GetKustoQuery(string query, string stampName)
+        {
+            return MakeDependencyCall(_kustoDataProvider.GetKustoQuery(query, stampName));
+        }
```

Comment:
Is it intentional that you want detector authors to be able to call this?

Reply:
Nevermind I just read your note on the top :)

Reply:
:-)